### PR TITLE
Scaffold page partners

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

**Comment by varffs** [comment]
Probably need to specify in the metafields which color the logos should be for each sponsors/partners section as for this one they can't be black:

screen shot 2016-07-13 at 15 38 16

But in the footer they are better black than white. And header cant be black either

**Comment by varffs** [comment]
Also this text `Material agradece el apoyo de los siguientes patrocinadores y colaboradores, sin los cuales esta feria no podría ser posible.` is hardcoded and not localized. Probably doesnt need to be metafield but does need to be english too

**Comment by mikeyrayvon** [comment]
oh thats an accident.  i was super tired

**Comment by mikeyrayvon** [comment]
ok should be all good @varffs

File: lib/theme-options/sponsors-options.php [hunk 1]
```diff
@@ -61,6 +62,15 @@
       'type' => 'title',
     ),
     array(
+      'name' => __( 'Partners Introduction Text', 'IGV' ),
+      'id'   => $prefix . 'program_page_text',
```
Comment (varffs) [inline lib/theme-options/sponsors-options.php]: this id should be `partners_page_text`

**Pull request merged**